apps/web/src/app/organizations/manage/groups.component.ts

```diff
@@ -200,23 +208,23 @@ export class GroupsComponent implements OnInit, OnDestroy {
     this.didScroll = this.pagedGroups.length > this.pageSize;
   }
 
-  async edit(groupRow: GroupDetailsRow) {
-    const [modal] = await this.modalService.openViewRef(
-      GroupAddEditComponent,
-      this.addEditModalRef,
-      (comp) => {
-        comp.organizationId = this.organizationId;
-        comp.groupId = groupRow != null ? groupRow.details.id : null;
-        comp.onSavedGroup.pipe(takeUntil(this.destroy$)).subscribe(() => {
-          modal.close();
-          this.refreshGroups$.next();
-        });
-        comp.onDeletedGroup.pipe(takeUntil(this.destroy$)).subscribe(() => {
-          modal.close();
-          this.removeGroup(groupRow.details.id);
-        });
+  edit(group: GroupDetailsRow, startingTabIndex: GroupAddEditTabType = GroupAddEditTabType.Info) {
+    const dialogRef = openGroupAddEditDialog(this.dialogService, {
+      positionStrategy: this.overlay.position().global().centerHorizontally(),
```

Comment:
issue: this blocks clicks on the backdrop for the whole width of the dialog. Make the following adjustment to fix :) 

```suggestion
      positionStrategy: this.overlay.position().global().centerHorizontally().top(),
```

Reply:
Fixed, nice catch!